rosdep/base.yaml

```diff
@@ -4133,6 +4138,7 @@ libogre-dev:
     artful: [libogre-1.9-dev]
     bionic: [libogre-1.9-dev]
     focal: [libogre-1.9-dev]
+    jammy: [libogre-1.9-dev]
```

Comment:
You missed my suggestion about turning this into a wildcard. However, actually looking at this all versions back to xenial are `[libogre-1.9-dev]` so we can just collapse this. to be an unversioned rule as they're all the same now.

Reply:
The same is true for `libogre` itself: https://packages.ubuntu.com/jammy/libogre-1.9.0v5
(libogre-dev: https://packages.ubuntu.com/jammy/libogre-1.9-dev)
Should I collapse this too?

Reply:
I have changed both, please tell me if I should change it :)